src/collective/solr/manager.py

```diff
@@ -1,67 +1,23 @@
 # -*- coding: utf-8 -*-
-from collective.solr.interfaces import ISolrConnectionConfig
 from collective.solr.interfaces import ISolrConnectionManager
 from collective.solr.interfaces import IZCMLSolrConnectionConfig
 from collective.solr.local import getLocal
 from collective.solr.local import setLocal
 from collective.solr.solr import SolrConnection
+from collective.solr.utils import getConfig
+from collective.solr.utils import isActive
 from httplib import CannotSendRequest
 from httplib import ResponseNotReady
 from logging import getLogger
-from persistent import Persistent
 from socket import error
-from zope.component import getUtility
 from zope.component import queryUtility
 from zope.interface import implements
+from plone.api.portal import get_registry_record
 
 logger = getLogger('collective.solr.manager')
 marker = object()
 
 
-class BaseSolrConnectionConfig(object):
```

Comment:
Do we really need this complexity? I mean it is just a connection between a CMS and an index based on an api (collective.indexing). It shouldn't be that hard. PloneIntranet moved away from collective.solr because of this complexity.